core/boostrenderer.py

```diff
@@ -50,6 +50,11 @@ def get_content_from_s3(key=None, bucket_name=None):
             response = client.get_object(Bucket=bucket_name, Key=s3_key.lstrip("/"))
             file_content = response["Body"].read()
             content_type = response["ContentType"]
+
+            # Check if the file ends with '.js', if yes then set the content_type to 'application/javascript'
+            if s3_key.endswith(".js"):
+                content_type = "application/javascript"
```

Comment:
That's pretty irritating, I thought this was handled automatically by the web server?

Reply:
@vinniefalco I think it is and this code is probably not needed. The real fix was in the config file. I tried this fix in https://github.com/cppalliance/temp-site/pull/328 but it didn't work, and since i re-used the same branch, I confused GitHub a little.